api/config/permissions.php

```diff
@@ -103,6 +120,17 @@
             PermissionsController::class.'@getAll',
         ],
     ],
+    'ManipulateWithOwnChild' => [
+        'type' => 'permission',
+        'description' => 'General permission to update record which belongs to the user',
+        'ruleName' => ManipulateWithOwnChild::class,
+        'children' => [
+            ArticleRateController::class.'@putOne',
+            ArticleBookmarkController::class.'@putOne',
+            ArticleRateController::class.'@deleteOne',
+            ArticleBookmarkController::class.'@deleteOne',
+        ],
+    ],
 
     //roles
```

Comment:
It's not in these changes, but in reviewing the whole file I notices that there is inconsistency in permission naming - some start with capital letter, others start with lowercase. Please fix all to start with capital (probably the best option to keep it in line with the class names they might match up to)